### ISSUE TITLE
[nativescript-stripe] confirmSetupIntent throws an error

### ISSUE THREAD

**Issue by sebj54** [triniwiz/nativescript-plugins#110]
Hi there!

I just updated the nativescript-stripe plugin to the latest version. It turns out that my implementation does not work anymore.

So far, the payment method creation still works, but when I want to confirm setup intent with:

```js
stripe.confirmSetupIntent(paymentMethod.id, this.clientSecret, onSuccessCallback)
```

I have this error:

```
TypeError: error.getMessage is not a function
```

It is thrown in this catch block: https://github.com/triniwiz/nativescript-plugins/blob/12f13aeb7361cb9c28554bcb90974587e5527084/packages/nativescript-stripe/index.android.ts#L660-L662

This happens because the error is not a Stripe error but a JS exception:

```
 Error: java.lang.NoSuchMethodError: no static method "Lcom/stripe/android/Stripe;.confirmSetupIntent(Landroidx/activity/ComponentActivity;Lcom/stripe/android/model/ConfirmSetupIntentParams;)V"
```

I wanted to debug this, but I'm a bit lost with Stripe Android documentation… There seems to be two `confirmSetupIntent` methods with a different signature:

- https://stripe.dev/stripe-android/payments-core/com.stripe.android/-stripe/index.html#-332170914%2FFunctions%2F-1622557690
- https://stripe.dev/stripe-android/payments-core/com.stripe.android/confirm-setup-intent.html

Thanks in advance for your help 🙏

**Comment by sebj54** [comment]
So I digged deeper to find a solution and can't find it. 

There is a method with the same signature (inspected with `java.util.Arrays.toString(this.stripe.getClass().getMethods())`): 

```java
public final void com.stripe.android.Stripe.confirmSetupIntent(androidx.activity.ComponentActivity,com.stripe.android.model.ConfirmSetupIntentParams)
```

Is this method working on your side?

**Comment by triniwiz** [comment]
What does `console.log(androidx.activity.ComponentActivity)` print ?

**Comment by sebj54** [comment]
It prints `function () { [native code] }`

**Comment by Stanteq** [comment]
I have the same problem on Android 10 :(

**Comment by triniwiz** [comment]
Can you try `9.0.0-alpha.0` version of stripe along with `8.4.0-alpha.0` of the android runtime

**Comment by Stanteq** [comment]
@triniwiz Unfortunately this (9.0.0-alpha.0) does not fix the problem (tested on Android 10 & 13).

**Comment by matteonatale** [comment]
Hi there, I'm having the same issue of @sebj54 and I'm in the same condition: after plugin update my implementation does not work either. Is there any news?

**Comment by sebj54** [comment]
Hi @triniwiz, I just tested the latest version (9.0.0-beta.1) and it still has the same problem. How can I help? Do you need a reproduction?

**Comment by sebj54** [comment]
The error seems fixed with latest beta (`9.0.0-beta.2`). Thanks @triniwiz!
